### ISSUE TITLE
TestFixture with TestName displayed twice in TestExplorer

### ISSUE THREAD

**Issue by goblinmaks** [nunit/nunit3-vs-adapter#729]
"NUnit" version="3.12.0" targetFramework="net45"
"NUnit.Engine.Api" version="3.11.1" targetFramework="net45"
"NUnit3TestAdapter" version="3.16.1" targetFramework="net45" developmentDependency="true" 

Same behavior on dotNetCore 3.1.100  

Microsoft Visual Studio Community 2019
Version 16.4.6

TestFixture with TestName displayed twice in TestExplorer, but Fixture that called as class name can't  be executed via TestExplorer. 

![image](https://user-images.githubusercontent.com/1436234/76904568-edb3cf00-68a8-11ea-8ffc-09f98b08c983.png)

**Comment by OsirisTerje** [comment]
Hi!  We have several issues related to custom testnames.  Some of these have been fixed in later versions of the adapter.  Can you please try version 3.16.1, or even the developer version of 3.17 (myget feed).  If still present, then please upload a small repro project .

**Comment by goblinmaks** [comment]
Still reproduced on NUnit3TestAdapter 3.17.0-dev-01365

[NUnitAdapterIssue729TestProject.zip](https://github.com/nunit/nunit3-vs-adapter/files/4348068/NUnitAdapterIssue729TestProject.zip)

**Comment by goblinmaks** [comment]
@OsirisTerje could you provide any tips how to find reason of this issue?
I have reviewed XML dump that uses in adapter during discovery and this one looks good without duplicates.

**Comment by OsirisTerje** [comment]
@goblinmaks Sorry for the late answer.   This is another of the FQN issues.  It is the RTD that comes in the way.  The output from NUnit is fine.  

What I see with RTD on is:
![image](https://user-images.githubusercontent.com/203432/84059528-0ec52100-a9bb-11ea-9252-6b04286822ac.png)

Turning off RTD gives this:
![image](https://user-images.githubusercontent.com/203432/84059596-2c928600-a9bb-11ea-8a91-33a2d983bb19.png)

I'll group this in with the other FQN issues.  This can only be solved together with the RTD.

**Comment by Youssef1313** [comment]
@OsirisTerje If this is caused only by real-time discovery and is still present today, I think it should be filed to Visual Studio. Did you file it already?

**Comment by OsirisTerje** [comment]
@Youssef1313  Yes, it is exactly the same.  I have tried now with the latest versions, and with VS 2026.   Where should it be filed?  Github or DevCom ?

**Comment by Youssef1313** [comment]
DevCom

**Comment by Laniusexcubitor** [comment]
[NUnit TestFixture with TestName shows up duplicated in Test Explorer (DevCom)](https://developercommunity.visualstudio.com/t/NUnit-TestFixture-with-TestName-shows-up/11053832)
